lib/fog/openstack/models/introspection/rules_collection.rb

```diff
@@ -0,0 +1,36 @@
+require 'fog/openstack/models/collection'
+require 'fog/openstack/models/introspection/rules'
+
+module Fog
+  module Introspection
+    class OpenStack
+      class RulesCollection < Fog::OpenStack::Collection
+        model Fog::Introspection::OpenStack::Rules
+
+        def all(_options = {})
+          load_response(service.list_rules, 'rules')
+        end
+
+        def summary(_options = {})
+          load_response(service.list_rules, 'rules')
+        end
+
+        def get(uuid)
+          data = service.get_rules(uuid).body
+          new(data)
+        rescue Fog::Introspection::OpenStack::NotFound
+          nil
+        end
+
+        def destroy(uuid)
```

Comment:
I understand the idea but I'm not following you 'cause the superclass only has method place holders. Are you willing to change that? Nevertheless if we change the superclass, I'd rather do that on a separate patch.